src/desktop/apps/auctions/templates/index.jade

```diff
@@ -15,7 +15,7 @@ block body
       .aggregate-page-items-featured
         if currentAuctions.length
           .aggregate-page-items-featured-set
-            h2.auctions-header Current Auctions
+            h1.auctions-header Current Auctions
```

Comment:
@eessex Is it bad from an SEO perspective to have multiple h1s? I'm noticing that there are [many on this page](https://github.com/artsy/force/blob/7bf1935125e3ae674d2c49912369e2b8df47f739/src/desktop/apps/auctions/templates/index.jade#L48-L49).

Down to merge assuming that we aren't making an existing SEO problem any worse @eessex

Reply:
Excellent point- will update those. Apparently we are no longer dinged for multiple h1’s, but we should be semantic either way and only use them for the page title.